src/main/kotlin/com/vk/admstorm/ui/AdmIcons.kt

```diff
@@ -3,7 +3,7 @@ package com.vk.admstorm.ui
 import com.intellij.ui.IconManager
 
 object AdmIcons {
-    private fun icon(name: String) = IconManager.getInstance().getIcon(name, javaClass)
+    private fun icon(name: String) = IconManager.getInstance().getIcon(name, javaClass.classLoader)
```

Comment:
why?

Reply:
.getIcon(name, javaClass) deprecated
I updated it

Reply:
ok